mw-webapp/src/component/header/Header.tsx

```diff
@@ -78,6 +74,10 @@ export const Header = () => {
     handleUserAuthState(setUser);
   }, []);
 
+  if (user) {
+    sessionStorage.setItem("user", user.uid);
+  }
```

Comment:
why do we need it?

Reply:
I need sessionStorage for save userUuid after login for render it's name after added comments/ That need because we use one column comments now instead of separate columns studentCmments and mentorComments

Reply:
get rid of it

Reply:
Used context